gallery/plot_visualization_utils.py

```diff
@@ -140,18 +136,34 @@ def show(imgs):
 # boat class:
 
 sem_classes = [
-    '__background__', 'aeroplane', 'bicycle', 'bird', 'boat', 'bottle', 'bus',
-    'car', 'cat', 'chair', 'cow', 'diningtable', 'dog', 'horse', 'motorbike',
-    'person', 'pottedplant', 'sheep', 'sofa', 'train', 'tvmonitor'
+    "__background__",
```

Comment:
Let's tell `black` to ignore the `gallery` folder. We don't want readers to have to scroll for such narrative-style documentation.

Reply:
You mean `isort`, right? Telling `black` ignore it will not make a difference for the imports.

Reply:
I meant `black` (as it can introduce a lot of vertical space), but we should avoid applying `isort` there too :)